### PR TITLE
Create issue templates

### PR DESCRIPTION
**Pull request by szuecs**
Add issue templates to guide user requests for enhancements and bug reports

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
```
Comment (aryszka) [inline .github/ISSUE_TEMPLATE/feature_request.md]: "Ex." as "excluding" or as "example"? Should it be "E.g."?

Comment (szuecs) [reply .github/ISSUE_TEMPLATE/feature_request.md]: Ah that was a default value :)

Comment (szuecs) [reply .github/ISSUE_TEMPLATE/feature_request.md]: changed a bit to be more clear

**Comment by aryszka** [comment]
:+1:

**Comment by szuecs** [comment]
:+1:

**Pull request merged**